react-app/src/library/services/application.ts

```diff
@@ -14,10 +14,10 @@ const hostedContext = {
   requestJabraDevices: (): Promise<any> => {
```

Comment:
I can't click it, but the `isHosted` could/should be determined by checking `window.cefQuery`. If it exists, then we are hosted.